code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -104,7 +104,7 @@
 //Applies brute and burn damage to the organ. Returns 1 if the damage-icon states changed at all.
 //Damage will not exceed max_damage using this proc
 //Cannot apply negative damage
-/obj/item/bodypart/proc/take_damage(brute, burn, updating_health = 1)
+/obj/item/bodypart/proc/receive_damage(brute, burn, updating_health = 1)
```

Comment:
changed the proc name since it doesn't do the same thing as /obj/proc/take_damage(). Maybe in the future we'll have the bodypart actually use a version of take_damage(), but for now we have two procs doing different things, so two names to avoid confusion.